lib/gems/pending/util/vmdb-logger.rb

```diff
@@ -157,7 +157,7 @@ class Formatter < Logger::Formatter
 
     def call(severity, time, progname, msg)
       msg = prefix_task_id(msg2str(msg))
-
+      msg = truncate_large_lines(msg)
```

Comment:
How about just `msg = prefix_task_id(msg2str(msg)).truncate(max_log_line_length)` ?

Reply:
Guess so, was worried about making an incomplete char at the end of the line, then again that could happen with `byteslice` too.

And I guess we're truncating the line anyway so a bust char wouldn't be the end of the world?

Reply:
yeah and it automatically appends `...` so you can tell that it was truncated